ant/apple/apple-keygen.sh.in

```diff
@@ -125,7 +125,7 @@ eval "$makedercert" > /dev/null 2>&1
 check_exists "$dercertpath"
 
 echo "Writing properties file..."
-echo "wss.alias=${jks.alias}" > "$propspath"
+echo "wss.alias=${jks.alias}" >> "$propspath"
```

Comment:
I think I understand the purpose of this as you can write `ca.certificate` prior to installing, right?  I don't like this approach as subsequent runs of the keygen utilities for Linux, Windows and Mac (such as in print server mode) will append to the bottom unless the installer is re-run.  I'd rather keep the existing behavior of writing out this file each time the script is run to guarantee it is a fresh file.